docs/bin/plugin_formatter.py

```diff
@@ -1,22 +1,9 @@
 #!/usr/bin/env python
-# (c) 2012, Jan-Piet Mens <jpmens () gmail.com>
-# (c) 2012-2014, Michael DeHaan <michael@ansible.com> and others
-# (c) 2017 Ansible Project
-#
-# This file is part of Ansible
-#
-# Ansible is free software: you can redistribute it and/or modify
-# it under the terms of the GNU General Public License as published by
-# the Free Software Foundation, either version 3 of the License, or
-# (at your option) any later version.
-#
-# Ansible is distributed in the hope that it will be useful,
-# but WITHOUT ANY WARRANTY; without even the implied warranty of
-# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-# GNU General Public License for more details.
-#
-# You should have received a copy of the GNU General Public License
-# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+# Copyright: (c) 2012, Jan-Piet Mens <jpmens () gmail.com>
+# Copyright: (c) 2012-2014, Michael DeHaan <michael@ansible.com> and others
+# Copyright: (c) 2017, Ansible Project
```

Comment:
Please change this to 2017‒2018, as the copyright should normally cover all years of artwork/code creation/updates.

Reply:
This has come up before, I'd rather get a consensus from the Documentation team, #45989

Reply:
excellent question, and one we should probably ask of someone with a legal background - I will find someone and ask, then report back on #45989

Reply:
Well, it's easily googlable :) I've shared links in some PR recently. Let me see whether I can find them

Reply:
Here you go https://smallbusiness.chron.com/year-copyright-statement-mean-62027.html

Reply:
Basically, copyright comes into play since content creation, so you need a year, when it's been originally created. When you update content, you "create" new parts of it, that requires new year to be mentioned.

Reply:
#45989 has a response on it.  The preferred way is to *not* advance the date or add a range.

Reply:
Thanks, that was quick !